tasties_app/tests/test_pytest.py

```diff
@@ -0,0 +1,2 @@
+def test_pass():
+    assert True
```

Comment:
We usually try to fail the CI first, so we can make sure it's doing what it's supposed to do

Reply:
Thank you for the important reminder. In this case, we know that our CI job can fail - since it did when we didn't have any tests defined, and the only purpose of this test_pytest.py file is to be a placeholder test until we define more tests.